<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="rpmBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>6000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>116</pointsize>
     </font>
    </property>
    <property name="text">
     <string>n/a</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="units">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string>km/h</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="soc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="text">
     <string>n/a%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="temperature">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="text">
     <string>n/a Â°C</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrent">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentTime">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>41</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>41</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Best:</string>
    </property>
   </widget>
   <widget class="QLabel" name="can">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Waiting for CAN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="errorCounter">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>No errors</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="power">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="text">
     <string>n/a kW</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
